spec/support/vacancy_helpers.rb

```diff
@@ -134,6 +134,7 @@ def verify_vacancy_list_page_details(vacancy)
     expect(page.find('.vacancy')).to have_content(vacancy.publish_on)
     expect(page.find('.vacancy')).to have_content(vacancy.starts_on)
     expect(page.find('.vacancy')).to have_content(vacancy.working_patterns)
+    expect(page.find('.vacancy')).to have_content(vacancy.expiry_time)
```

Comment:
It could be sensible to add a similar assertion to `verify_all_vacancy_details` and `verify_vacancy_show_page_details` tests